sdk-base/src/main/java/com/mapbox/maps/plugin/viewport/experimental/state/ViewportDataSourceObserver.kt

```diff
@@ -0,0 +1,16 @@
+package com.mapbox.maps.plugin.viewport.experimental.state
+
+import com.mapbox.maps.CameraOptions
+
+/**
+ * Observer that gets notified whenever new data is available.
+ */
+fun interface ViewportDataSourceObserver {
```

Comment:
Is this still necessary?

Reply:
this is for `fun observeDataSource` inside the ViewportState, we can discuss the naming here though.